test/integration/query-interface.test.js

```diff
@@ -112,6 +113,114 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
         expect(tableNames).to.include('my_test_table2');
       });
     }
+
+    if (dialect.supports.schemas) {
+      describe('schema support tests', () => {
+        const getSequelizeInstanceWithSchema = async () => {
+          const sequelize = createSequelizeInstance({ schema: 'schema_3' });
+          await sequelize.createSchema('schema_3');
+          await sequelize.queryInterface.createTable('schema_3_table_1', { name: DataTypes.STRING }, {
+            schema: 'schema_3',
+          });
+          await sequelize.queryInterface.createTable('schema_3_table_2', { name: DataTypes.STRING }, {
+            schema: 'schema_3',
+          });
+
+          await sequelize.createSchema('schema_4');
+          await sequelize.queryInterface.createTable('schema_4_table_1', { name: DataTypes.STRING }, {
+            schema: 'schema_4',
+          });
+          await sequelize.queryInterface.createTable('schema_4_table_2', { name: DataTypes.STRING }, {
+            schema: 'schema_4',
+          });
```

Comment:
Some of these operations can be run concurrently (creating both schema concurrently, then all tables)

These could also call `createTestTablesForSchema` instead of duplicating it

Reply:
Please review [this](https://github.com/sequelize/sequelize/pull/15101/files#diff-09e7808facf24ebfd40883f8804d029e3a984f3c447930790cc05ae919fc147cR13-R64) section again.